src/stats/stats_reader.cpp

```diff
@@ -190,6 +190,7 @@ void update_delta_stat(socket_stats_t* p_curr_stat, socket_stats_t* p_prev_stat)
 	p_prev_stat->counters.n_rx_poll_miss = (p_curr_stat->counters.n_rx_poll_miss - p_prev_stat->counters.n_rx_poll_miss) / delay;
 	p_prev_stat->counters.n_rx_poll_hit = (p_curr_stat->counters.n_rx_poll_hit - p_prev_stat->counters.n_rx_poll_hit) / delay;
 	p_prev_stat->n_rx_ready_byte_count = p_curr_stat->n_rx_ready_byte_count;
+	p_prev_stat->n_tx_ready_byte_count = p_curr_stat->n_tx_ready_byte_count;
```

Comment:
What is "ready" in the statistics of unacked packets denoted by the name: "n_tx_ready_byte_count"?
Please rename "n_tx_ready_byte_count" member name to "n_tx_unacked_byte_count"